swarm/network/stream/stream.go

```diff
@@ -108,7 +107,7 @@ type RegistryOptions struct {
 }
 
 // NewRegistry is Streamer constructor
-func NewRegistry(localID enode.ID, delivery *Delivery, syncChunkStore chunk.FetchStore, intervalsStore state.Store, options *RegistryOptions, balance protocols.Balance) *Registry {
+func NewRegistry(localID enode.ID, delivery *Delivery, netStore *storage.NetStore, intervalsStore state.Store, options *RegistryOptions, balance protocols.Balance) *Registry {
```

Comment:
RetrievalOption type, its constants, Registry.autoRetrieval, RegistryOptions.Retrieval and references in tests can be removed, since this values are not used anymore.

Reply:
Is it okay to do that in a follow up PR, or you want it done here?

Reply:
Since they are all related to the retrieval which is removed in this PR, I think that they should be removed here, too.

Reply:
removed.

Reply:
Thanks Anton